serde/compat.py

```diff
@@ -124,7 +137,7 @@ def get_origin(typ: Type[Any]) -> Optional[Any]:
     try:
         return typing.get_origin(typ) or get_np_origin(typ)  # python>=3.8 typing module has get_origin.
     except AttributeError:
-        return typing_inspect.get_origin(typ) or get_np_origin(typ)
+        return typing_extensions.get_origin(typ) or get_np_origin(typ)
```

Comment:
Because `typing_inspect.get_origin(ClassVar[int])` returns `None` instead of `ClassVar`.